src/Shift-Changes.package/ShAbstractChange.class/instance/^equals.st

```diff
@@ -0,0 +1,5 @@
+comparing
+= another
```

Comment:
what about more readable
	^ (self species = another species)
		and: [ builder = another builder ]